js/ui/remoteSearch.js

```diff
@@ -73,6 +73,10 @@ function loadRemoteSearchProviders(searchSettings, callback) {
     function loadRemoteSearchProvider(file) {
         let keyfile = new GLib.KeyFile();
         let path = file.get_path();
+        let evergreenApps = [
+            'com.endlessm.quote_of_the_day.en.desktop',
+            'com.endlessm.word_of_the_day.en.desktop'
+        ];
```

Comment:
Instead of hardcoding this list, you can simply use `appInfo.should_show()` and exclude the app if that's false. (It will look at the `NoDisplay` key in the desktop file behind the scenes.)